packages/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -77,7 +77,7 @@ export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
     getInitialState() ?? [
       // TODO(thure): Initialize these dynamically.
       ['root', create({ open: true, current: false })],
-      ['root~dxos.org/plugin/space-spaces', create({ open: true, current: false })],
+      ['root~dxos.org/plugin/space/spaces', create({ open: true, current: false })],
```

Comment:
you can't change this without changing the actual id of the graph node

Reply:
https://github.com/dxos/dxos/blob/main/packages/plugins/plugin-space/src/util.tsx#L40

Reply:
the state is cached so it would only break on fresh profile

Reply:
https://github.com/dxos/dxos/blob/main/packages/plugins/plugin-space/src/util.tsx#L40

Reply:
ok. i'll revert, but let's push to enforce consistency.